core/src/main/java/com/linecorp/armeria/common/StringMultimap.java

```diff
@@ -778,6 +778,7 @@ final void set(IN_NAME name, Iterable<String> values) {
             requireNonNullElement(values, v);
             addAndNotify(h, i, normalizedName, v, false);
         }
+        onChange(normalizedName);
```

Comment:
Sorry, it should be reverted. I added it to invalidate `contentLengthUnknown` flag when `CONTENT_LENGTH` is added.
The implementation has changed to `contentLengthMaybeUnknown` and it had to be reverted.
But I partially reverted the changes because I thought there was a bug that `onChange()` isn't called in this method. I missed `removeAndNotify()` at that time.